upgrade/v2.5/README.md

```diff
@@ -0,0 +1,48 @@
+# Calico v2.5.0 Upgrade Procedure
+
+## Why is this required
```

Comment:
nit, suggest "Why this is required" instead, or if you like the question format, append a "?"